### DISCUSSION TITLE
Pro Variables to allow Relationship fields

### DISCUSSION THREAD

**Discussion by agun** [ExpressionEngine/ExpressionEngine#2928] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I often need to create a site wide footer link group (very common on sites these days). Previously in Low Variables I would great a Grid field, and then use a Playa relationship field within the grid. Then an author can create one or more 'lists' of entries to which I can add into the footer (ie About us, with a list of About us entries, then a Services, with a list of Services entries, etc). Now that Pro Variables in built into EE, it would be great that we do not have to use Playa, but instead use the native Relationship field.

**Describe the solution you'd like**
I would like to see the **Relationship field** to be enabled for use in **Pro Variables**, as both **a stand alone field**, and also to be used in a Pro Variables **Grid field**.

**Describe alternatives you've considered**
Currently there is a "Select Entries" field that allows a stand-alone style relationship field (it's very clunky!) ... at minimum it would be good to be able to use this in a Grid field in Pro Variables. This is not my preference, but it is an alternative.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
Related to comment: https://github.com/ExpressionEngine/ExpressionEngine/issues/2498#issuecomment-1318723006 - @TomJaeger FYI

**Comment by intoeetive** [comment]
Noting that the way Relationship field is built it requires Channel Entry as parent to function correctly.
